lib/Queries/ReadQueries.js

```diff
@@ -197,7 +192,7 @@ class ReadQueries extends Queries {
                 1,
                 reference.perPage(),
                 typeof(filters) === 'function' ? filters(search) : filters,
-                null,
+                reference,
```

Comment:
You are true. Fixed, even if we always passed an empty object before. I passed an argument for better consistency.